Description: Set default polling interval on CD drives as well. The events_dfl_poll_msecs will not trigger if "block" is not a module, but built in. This will avoid udisks etc. having to poll from userspace, and provide proper ejection when the hardware eject button is pressed.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/890592

Index: precise/rules/rules.d/60-persistent-storage.rules
===================================================================
--- precise.orig/rules/rules.d/60-persistent-storage.rules	2012-02-06 09:09:39.413371483 +0100
+++ precise/rules/rules.d/60-persistent-storage.rules	2012-03-28 11:33:05.900794721 +0200
@@ -10,6 +10,7 @@
 
 # enable in-kernel media-presence polling
 ACTION=="add", SUBSYSTEM=="module", KERNEL=="block", ATTR{parameters/events_dfl_poll_msecs}=="0", ATTR{parameters/events_dfl_poll_msecs}="2000"
+ACTION=="add", ATTR{removable}=="1", ATTR{events_poll_msecs}=="-1", ATTR{events_poll_msecs}="2000"
 
 SUBSYSTEM!="block", GOTO="persistent_storage_end"
 
